projects/storefrontlib/src/lib/outlet/outlet-ref/outlet-ref.directive.ts

```diff
@@ -20,3 +28,21 @@ export class OutletRefDirective implements OnInit {
     this.outletService.add(this.cxOutletRef, this.tpl, this.cxOutletPos);
   }
 }
+
+@Directive({
```

Comment:
my intend wasn't to have both directives in one file. It's not wrong, but I'd prefer to separate them out in 2 files